### PR TITLE
Issue/1863 mysql sink quote fields

### PR DESCRIPTION
**Pull request by pmo-sdr**
Fix for Issue https://github.com/telefonicaid/fiware-cygnus/issues/1863

### CODE REVIEW

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java [hunk 1]
```diff
@@ -338,7 +338,7 @@ protected NGSIGenericAggregator getAggregator(boolean rowAttrPersistence) {
     private void persistAggregation(NGSIGenericAggregator aggregator)
```
[the remark this answers is not in the document]
Comment (fgalan) [reply cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java:338]: Fixed in 9010472

**Comment by IvanHdzC** [comment]
Great @dmartinezgomez. In general I think everything is alright.

I just realized that those SQL methods have no tests so, can you please add a couple of tests for this overloaded method. It would be easier to add them to NGSIUtilsTest class, not directly to the Sink.

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java [hunk 2]
```diff
@@ -48,7 +48,8 @@
  */
 public class NGSIMySQLSink extends NGSISink {
     
-    private static final String DEFAULT_ROW_ATTR_PERSISTENCE = "row";
+    private static final String MYSQL_QUOTE_CHAR = "`";
+	private static final String DEFAULT_ROW_ATTR_PERSISTENCE = "row";
```
Comment (fgalan) [inline cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java:52]: I have just spotted a little missindent in this line...

**Comment by fgalan** [comment]
> I just realized that those SQL methods have no tests so, can you please add a couple of tests for this overloaded method. It would be easier to add them to NGSIUtilsTest class, not directly to the Sink.

I see recent activity in the PR (auther by @FranGomezCiC on August 27th)... should we understand that the above point has been covered by the new modifications?

**Comment by joelcamus** [comment]
> 
> 
> > I just realized that those SQL methods have no tests so, can you please add a couple of tests for this overloaded method. It would be easier to add them to NGSIUtilsTest class, not directly to the Sink.
> 
> I see recent activity in the PR (auther by @FranGomezCiC on August 27th)... should we understand that the above point has been covered by the new modifications?

 We have added two tests for these SQL methods

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java (hunk 2 above)
Comment (joelcamus) [reply cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java]: done

**Comment by fgalan** [comment]
> We have added two tests for these SQL methods

Great! Could you solve the conflict on CHANGEX_NEXT_RELEASE so we can have a final look and eventually merge, please?

**Review by IvanHdzC — COMMENTED** [review]
Please update your branch with current master before solving merge conflicts.

**Comment by joelcamus** [comment]
> 
> 
> > We have added two tests for these SQL methods
> 
> Great! Could you solve the conflict on CHANGES_NEXT_RELEASE so we can have a final look and eventually merge, please?

Done

**Comment by joelcamus** [comment]
> 
> 
> Please update your branch with current master before solving merge conflicts.

Done

**Comment by joelcamus** [comment]
I apologise for the late

**Review by IvanHdzC — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
Thanks for the contribution!

**Pull request merged**